scalyr_agent/scalyr_client.py

```diff
@@ -937,6 +946,35 @@ def __get_user_agent(self, agent_version, fragments=None):
             # Indicates agent running as a regular user
             user_string = "admin-0"
 
+        sharded_copy_manager_string = ""
+
+        # Possible values for this header fragment:
+        # mw-0 - Sharded copy manager functionality is disabled
+        # mw-1|<num_workers>|<num_api_keys> - Functionality is enabled and there are <num_workers>
```

Comment:
Why use a `|` as a separator?  That feels more like it marks the beginning of new concept (i.e., we are transitioning now to information about K8s version).  My preference would just to use a hyphen to separate it.  But, I also won't fight for it.

Reply:
Either way works for me.

I thought we used ``-`` for separating main fragment key and value for for "sub-values" I should use something else, but I can also just use ``-`` for those values as well.

Reply:
I think we didn't specify a good policy to begin with.  Main fragments are separated by semi-colons... and that's about all we have.  Eh, I don't mind either way.